src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/callback/CreateCaseValidator.java

```diff
@@ -158,8 +159,8 @@ private ScannedDocument mapScannedDocument(Map<String, Object> document) {
             ((Map<String, String>) document.get("url")).get("document_url"),
             (String) document.get("controlNumber"),
             (String) document.get("fileName"),
-            Instant.parse((String) document.get("scannedDate")),
-            Instant.parse((String) document.get("deliveryDate"))
+            Instant.from(FORMATTER.parse((String) document.get("scannedDate"))),
+            Instant.from(FORMATTER.parse((String) document.get("deliveryDate")))
```

Comment:
no change in tests ? (or new tests)

Reply:
let me have a look. don't want to just add tests which cover similar functionality with integration. i have success path unit tests handy amend to cover such cases